piccolo/apps/user/commands/list.py

```diff
@@ -0,0 +1,26 @@
+from piccolo.apps.user.tables import BaseUser
+
+
+def list():
+    """
+    List existing users.
+    """
+    users = BaseUser.select(
+        BaseUser.all_columns(exclude=[BaseUser.password])
+    ).run_sync()
```

Comment:
I'd probably make this `async`.

Reply:
I've checked other `user` subcommands and they are synchronous.

Reply:
You're right - not a big problem, lets leave it.